### DISCUSSION TITLE
Love it!

### DISCUSSION THREAD

**Discussion by johnwoltman** [lllyasviel/Omost#90] [General]
I've only just installed it and made a few images, but it's a really neat tool.  Thanks @lllyasviel, for all the hard work you do for the stable diffusion community :D

**Comment by Lustwaffel** [comment]
Yup. It's awesome. It takes time but is so much better at producing things that I would LIKE to see. WIthout having to prepare the whole thing in advance with sketches and 3d. 
Though I would just love to be able to switch models withouth editing the backend and other such common features. But MOST important would be to be able to incorporate all the controlnet features. The rest is trivial. This is a wonderful, powerful tool for composition and actually making images that are not just copies of other peoples prompts.
